src/components/TrackTile/TrackerDetails/TrackerHistoryChart/Chart.tsx

```diff
@@ -134,93 +77,123 @@ export const Chart: FC<ChartProps> = (props) => {
             </Text>
           </View>
         ))}
-      </ChartContent>
+      </View>
+      <View style={styles.barsContainer}>
+        {days.map((day, index) => (
+          <View
+            style={variantStyles?.chartValueBar}
+            key={`${day}-bar-container`}
+          >
+            {/* Active Bar */}
+            {!loading && !hasError && (
+              <Bar
+                variant={variant}
+                style={!isToday(day) ? { barFlat: { opacity: 0.35 } } : {}}
+                color={darkenHexColor(
+                  color,
+                  isToday(day) || isDefault ? 0 : 45,
+                )}
+                testID={tID(`history-chart-value-bar-${index}`)}
+                percentComplete={
+                  values[index] / ticksMax >= 1 ? 1 : values[index] / ticksMax
```

Comment:
I think you might be able to drop all of the d3 scale and tick stuff since we always display the bars based on the target now and they never exceed that width. Although this might be handling how we handle a target of zero. Not really sure how that works.